website/static/js/profile.js

```diff
@@ -260,14 +277,21 @@ BaseViewModel.prototype.handleSuccess = function() {
     }
 };
 
-BaseViewModel.prototype.handleError = function(response) {
+BaseViewModel.prototype.handleError = function(response) {    
     var defaultMsg = 'Could not update settings';
-    var msg = response.responseJSON.message_long || defaultMsg;
-    this.changeMessage(
-        msg,
-        'text-danger',
+    var msg;
+    if (typeof response.responseJSON != "undefined") {
```

Comment:
As long as you're setting `dataType: json` for all AJAX calls, `response` should be the parsed JSON, and you won't need to access `response.responseJSON`. 

Note: I believe the code that was here before was incorrect.